plugins/woocommerce/client/blocks/assets/js/blocks/mini-cart/iapi-frontend.ts

```
@@ -245,6 +268,26 @@ store< MiniCart >(
+			get contentsBackgroundColor(): string {
+				return (
+					getComputedStyle( document.body ).backgroundColor || '#fff'
+				);
+			},
+
+			get badgeBackgroundColor(): string | undefined {
+				if ( state.isHydrated ) {
+					const { ref } = getElement();
+					return getClosestColor( ref!, 'color' ) || '#000';
+				}
+			},
```

Biome (2.3.13)

[error] This getter should return a value.



(lint/suspicious/useGetterReturn)